demo/demo-netgenerate.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e
+
+source venv/bin/activate
+export SUMO_HOME=/home/user/projects/uni/d610f20/SUMO
+
+rm "out/cities/*" || true
+
+echo "====================="
+echo ""
+echo " NETGENERATE "
+echo ""
+echo "====================="
+
+./in/generateNetLinux.sh 
```

Comment:
`generateNetLinux.sh` doesn't use the utility from the a docker contains, and instead assumes you have the tool in `SUMO_HOME/bin/netgenerate`

Reply:
That's hard for me to test, since it depends on your setup.